modules/maliput_malidrive/0.3.1/presubmit.yml

```diff
@@ -0,0 +1,23 @@
+matrix:
+  bazel: [6.x, 7.x]
+  platform: ["ubuntu2004"]  # "debian10", "macos", "windows"
+tasks:
+  verify_targets:
+    name: Verify Build Targets
+    bazel: ${{ bazel }}
+    platform: ${{ platform }}
+    build_targets:
```

Comment:
Can you add `--process_headers_in_dependencies` like the boost libraries?

https://github.com/bazelbuild/bazel-central-registry/blob/c446821ee486cfee60a0a82e990d9dec6bd1eb1e/modules/boost.functional/1.83.0.bcr.1/presubmit.yml#L16-L17

It'll help actually verify the cc_library target

Reply:
Done at 3f42bb5